boot/mbed/CMakeLists.txt

```diff
@@ -33,18 +33,16 @@ target_link_libraries(${LIB_TARGET}
         mbed-mbedtls
         mbed-storage-flashiap
         mbed-storage-blockdevice
+        mbed-core-flags
 )
 
-if("_RTE_" IN_LIST MBED_CONFIG_DEFINITIONS)
-    target_link_libraries(${LIB_TARGET}
-    PUBLIC
-        mbed-os
-    )
-else()
-    target_link_libraries(${LIB_TARGET}
-    PUBLIC
-        mbed-baremetal
-    )
+# Add signing key generated source file
+mcuboot_generate_signing_keys_file(${CMAKE_CURRENT_BINARY_DIR}/signing_keys.c)
+target_sources(${LIB_TARGET} PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/signing_keys.c)
+
+if("MCUBOOT_ENCRYPT_RSA=1" IN_LIST MBED_CONFIG_DEFINITIONS)
+	mcuboot_generate_encryption_key_file(${CMAKE_CURRENT_BINARY_DIR}/enc_keys.c)
+	target_sources(${LIB_TARGET} PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/enc_keys.c)
```

Comment:
2 space indent for cmake

Reply:
Updated.  Also could you approve the workflows if you have a sec?